settings.gradle

```diff
@@ -1,4 +1,4 @@
-include ':step00-start'
+include ':step00-start', ':shared'
```

Comment:
For multiple module project like this where steps can be commented out, etc, prefer to have each on their own line. Shared should go first to mimic the dependency tree.